custom/workflow-helper/app-deployer/app-manifest/resilient-podtato-head.yaml

```diff
@@ -22,7 +22,7 @@ spec:
       terminationGracePeriodSeconds: 5
       containers:
       - name: server
-        image:  gdsoumya/podtato-main-js:v0.1.0
+        image:  litmuschaos/podtato-main-js:v0.1.0
```

Comment:
where are we preparing these images?

Reply:
This is the repo: https://github.com/gdsoumya/podtato-head/tree/custom-chaos
It contains the scripts for building and pushing the images to DockerHub. We will be maintaining this too, can you create an empty repo so that I can add the files there?